spec/models/article_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  let(:article) {create(:article)}
+
+  describe 'Database table' do
+    it { is_expected.to have_db_column :headline}
+    it { is_expected.to have_db_column :content}
```

Comment:
Mind the code formatting. you have a space after the opening curly but not before the enclosing curly.